### PR TITLE
Fe feature add bouquet mobile and tablet

### PR DESCRIPTION
**Pull request by halynarydasova**

### CODE REVIEW

File: frontend/src/redux/slices/cartSlice.js [hunk 1]
```diff
@@ -9,28 +9,99 @@ const cartSlice = createSlice({
...
         state[type + 'Items'][itemIndex].cardQuantity += 1;
+        saveToLocalStorige(
+          info.id,
+          state[type + 'Items'][itemIndex].cardQuantity + 1,
+        );
       } else {
         const tempProduct = { ...info, cardQuantity: 1 };
         state[type + 'Items'].push(tempProduct);
       }
+      saveToLocalStorige(info.id);
```
Comment (Svitlana-G) [inline frontend/src/redux/slices/cartSlice.js:58]: Can we simplify `addToCart`? For example so:
```
`    addToCart(state, action) {
      const { info, type } = action.payload;
      const itemIndex = state[type + 'Items'].findIndex((item) => item.id === info.id);
      if (itemIndex >= 0) {
        state[type + 'Items'][itemIndex].cardQuantity += 1;
      } else {
        state[type + 'Items'].push({ ...info, cardQuantity: 1 });
      }
      saveToLocalStorage(type, state[type + 'Items']);
    `}`
```

File: frontend/src/components/Layouts/Header/components/TotalItems/TotalItems.module.scss [hunk 2]
```diff
@@ -0,0 +1,35 @@
+@import '/src/variables';
+@import '/src/mixins';
+
+.cartQuantity {
+  padding: 1px 5px 0;
+  font-family: 'FixelDisplay';
+  border-radius: 100px;
+  background-color: #de5f38;
```
[the remark this answers is not in the document]
Comment (halynarydasova) [reply frontend/src/components/Layouts/Header/components/TotalItems/TotalItems.module.scss:8]: done

File: frontend/src/redux/slices/cartSlice.js [hunk 3]
```diff
@@ -9,28 +9,99 @@ const cartSlice = createSlice({
   name: 'cart',
   initialState,
   reducers: {
+    initiateCart(state, action) {
+      const { type, items } = action.payload;
+      state[type + 'Items'] = items;
+    },
     addToCart(state, action) {
+      const saveToLocalStorige = (id) => {
```
[the remark this answers is not in the document]
Comment (halynarydasova) [reply frontend/src/redux/slices/cartSlice.js:17]: done

File: frontend/src/redux/slices/cartSlice.js [hunk 4]
```diff
@@ -9,28 +9,99 @@ const cartSlice = createSlice({
...
+          );
+        } else {
+          const newItem = { id: id, cardQuantity: 1 };
+          items.push(newItem);
+        }
+        try {
+          localStorage.setItem(type, JSON.stringify(items));
+        } catch (error) {
+          console.error('Error saving to localStorage:', error);
+        }
```
[the remark this answers is not in the document]
Comment (halynarydasova) [reply frontend/src/redux/slices/cartSlice.js:41]: done

**Pull request merged**